config/containers/logging/json-file.md

```diff
@@ -27,22 +23,17 @@ configuring Docker using `daemon.json`, see
 [daemon.json](/engine/reference/commandline/dockerd.md#daemon-configuration-file).
 
 The following example sets the log driver to `json-file` and sets the `max-size`
-and `max-file` options.
+option.
 
 ```json
 {
   "log-driver": "json-file",
   "log-opts": {
-    "max-size": "10m",
-    "max-file": "3"
+    "max-size": "10m"
   }
 }
 ```
 
-> **Note**: `log-opt` configuration options in the `daemon.json` configuration
-> file must be provided as strings. Boolean and numeric values (such as the value
-> for `max-file` in the example above) must therefore be enclosed in quotes (`"`).
```

Comment:
Why was this removed? Is this because of a bad rebase?

Reply:
Added back in.